src/FbTk/Container.cc

```diff
@@ -393,15 +393,15 @@ void Container::repositionItems() {
                 // "a" and "a very long item with useless information" won't
                 // become "a very long item with" and ""
                 overhead += buttonDemands.size(); // compensate forrounding errors
-                const int mean = totalDemands / buttonDemands.size();
-                const int thresh = 3 * mean / 2;
+                const unsigned int mean = totalDemands / buttonDemands.size();
```

Comment:
why the switch to `unsigned int`?

Reply:
Don't know; didn't come up in #76 this time.